Caddyfile

```diff
@@ -12,7 +12,7 @@
 	# by uncommenting the line below
 	#
 	# auto_https off
-	# admin 0.0.0.0:2019
+	admin 0.0.0.0:2019
```

Comment:
We need this only for local development. Adding a reminder in the readme to uncomment this should be good.

Reply:
fixed

Reply:
```suggestion
	# admin 0.0.0.0:2019
```